### PR TITLE
Added: Disk number for tracks

### PR DESCRIPTION
**Pull request by danielunderwood**
Return disk number and relative track number instead of absolute track number.

### CODE REVIEW

File: lidarrmetadata/sql/track_album_mbid.sql [hunk 1]
```diff
@@ -1,14 +1,15 @@
-SELECT DISTINCT
+SELECT
   track.id,
   track.gid,
   track.name,
   track.position,
-  recording.length
+  track.number,
+  track.length,
```
Comment (Qstick) [inline lidarrmetadata/sql/track_album_mbid.sql:7]: I suppose there is no difference between length in track table and recording table? This is better due to join elimination, just making sure.

Comment (danielunderwood) [reply lidarrmetadata/sql/track_album_mbid.sql:7]: I'm going to have to dig into it a bit more. The following query does return some results, but it's mostly that the recording is longer by a second or so (and up to 5 for one that I saw). Doing `WHERE recording.length < track.length` does also return some results. I suppose it comes up to the difference of a track and a recording.
```
SELECT
  track.id,
  track.gid,
  track.name,
  track.length,
  recording.length
FROM track
  JOIN recording on track.recording = recording.id
WHERE recording.length != track.length
LIMIT 10
```

Comment (Qstick) [reply lidarrmetadata/sql/track_album_mbid.sql:7]: Ok, if most are just a second or so it shouldn’t be an issue. I was just concerned if it’s a large difference or if example track.length wasn’t filled out as commonly as recording.length or vise versa. Probably is quicker without the extra join

File: lidarrmetadata/provider.py [hunk 2]
```diff
@@ -471,11 +487,14 @@ class MusicbrainzDbProvider(Provider,
                             ArtistLinkMixin,
                             ArtistNameSearchMixin,
                             AlbumByArtistMixin,
+                            MediaByAlbumMixin,
                             TracksByAlbumMixin):
     """
     Provider for directly querying musicbrainz database
     """
 
+    TRACK_NUMBER_REGEX = re.compile(r'(?P<disk>[A-Z]*)(?P<track>\d+)')
```
Comment (Qstick) [inline lidarrmetadata/provider.py]: Is this still needed/used?

**Review by Qstick — APPROVED** [review]

**Pull request merged**